src/Umbraco.Web.UI.Client/src/views/common/dashboard.controller.js

```diff
@@ -21,10 +23,9 @@ function DashboardController($scope, $routeParams, dashboardResource, localizati
     
     dashboardResource.getDashboard($routeParams.section).then(function(tabs){
         $scope.dashboard.tabs = tabs;
-        
-        // set first tab to active
-        if($scope.dashboard.tabs && $scope.dashboard.tabs.length > 0) {
-            $scope.dashboard.tabs[0].active = true;
+
+        if ($scope.dashboard.tabs && $scope.dashboard.tabs.length > 0) {
+            initActiveTab();
         }
 
         $scope.page.loading = false;
```

Comment:
This _might_ remove the loading indicator before the localization service promise has succeeded. It's best to add/push all promises to an array and wait for all to succeed:

```js
// Add $q to the controllers constructor!

var promises = [];

promises.push(localizationService.localize("sections_" + $routeParams.section).then(function(name) {
    $scope.dashboard.name = name;
}));

// Same for dashboardResource.getDashboard

$q.all(promises).then(function () {
    $scope.page.loading = false;
}
```

If the un-localized name is not hidden while the page is loading, the indicator/overlay might need to be moved to do so (otherwise you could end up seeing the `[dashboard_name]` format). This is something I've also used/fixed in this PR: https://github.com/umbraco/Umbraco-CMS/pull/8384/files#diff-c9f6a6eb26104b4a81f7e06b9c6f2cdaf563a0854822413f1ca2083a15c6a066R73.